packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -27,6 +27,10 @@ export const OdieSendMessageButton = () => {
 		setIsMessageSizeValid( true );
 	}, [] );
 
+	useCallback( () => {
+		setIsChatBusy( chat.status === 'loading' || chat.status === 'sending' );
+	}, [ chat.status ] );
```

Comment:
This doesn't do anything. And I think statelessness is always better.

Reply:
Checking now
I was dealing with lots of issues on the PR and this might be a leftover that I didn't revert.

Reply:
Correct, that change was useless.
Reverted it now